### PR TITLE
Feature/content approval workflow

### PR DESCRIPTION
**Pull request by nitheshkumar-hcl**
Committing the changes for content approval workflow story

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            <metaData jcr:primaryType="nt:unstructured"/>
+        </node3>
+        <node4
+            jcr:primaryType="cq:WorkflowNode"
+            description="Rejecting the content as the content is not valid"
+            title="Reject the content"
+            type="PARTICIPANT">
+            <metaData
+                jcr:primaryType="nt:unstructured"
+                PARTICIPANT="content-author"/>
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml]: @nitheshkumar-hcl  update group name and append HCLecomm in name.

File: ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                        jcr:createdBy="admin"
+                        jcr:description="Rejecting the content as the content is not valid"
+                        jcr:lastModified="{Date}2020-06-16T18:29:21.024+05:30"
+                        jcr:lastModifiedBy="admin"
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Reject the content"
+                        sling:resourceType="cq/workflow/components/model/participant">
+                        <metaData
+                            jcr:primaryType="nt:unstructured"
+                            PARTICIPANT="content-author"/>
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml]: @nitheshkumar-hcl   update group name and append HCLecomm in name.

File: ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:designPath="/libs/settings/wcm/designs/default"
+        cq:lastModified="{Date}2020-06-16T19:01:59.971+05:30"
+        cq:lastModifiedBy="admin"
+        cq:template="/libs/cq/workflow/templates/model"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Content Approval Workflow"
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml]: @nitheshkumar-hcl  update group name and append HCLecomm in name.

File: ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml [hunk 4]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:isCheckedOut="{Boolean}false"
+    jcr:primaryType="cq:WorkflowModel"
+    jcr:uuid="72e1b6ff-d73e-4a66-b6c6-354718451e54"
+    sling:resourceType="cq/workflow/components/model"
+    description="No Description"
+    title="Content Approval Workflow">
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml]: @nitheshkumar-hcl  update group name and append HCLecomm in name.

File: ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml (hunk 1 above)
Comment (nitheshkumar-hcl) [reply ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml]: updated the workflow as per the review comments

File: ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml (hunk 4 above)
Comment (nitheshkumar-hcl) [reply ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml]: updated the workflow as per the review comments

File: ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml (hunk 3 above)
Comment (nitheshkumar-hcl) [reply ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml]: updated the workflow as per the review comments

File: ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml (hunk 2 above)
Comment (nitheshkumar-hcl) [reply ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml]: updated the workflow as per the review comments

File: ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml [hunk 5]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:isCheckedOut="{Boolean}false"
+    jcr:primaryType="cq:WorkflowModel"
+    jcr:uuid="72e1b6ff-d73e-4a66-b6c6-354718451e54"
+    sling:resourceType="cq/workflow/components/model"
+    description="No Description"
+    title="HCLEcomm Content Approval Workflow">
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml]: @nitheshkumar-hcl  provide description  and e should be in small in HCLEcomm

File: ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml [hunk 6]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:designPath="/libs/settings/wcm/designs/default"
+        cq:lastModified="{Date}2020-06-19T12:49:24.498+05:30"
+        cq:lastModifiedBy="admin"
+        cq:template="/libs/cq/workflow/templates/model"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="HCLEcomm Content Approval Workflow"
```
Comment (rakeshkumar-verma) [inline ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml]: @nitheshkumar-hcl  e should be in small in HCLEcomm

Comment (nitheshkumar-hcl) [reply ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml]: updated the workflow as per the review comments

File: ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml (hunk 5 above)
Comment (nitheshkumar-hcl) [reply ui.content/src/main/content/jcr_root/var/workflow/models/hclecomm/content-approval-workflow.xml]: updated the workflow as per the review comments

**Pull request merged**